src/repos/FeedRepository.ts

```
@@ -70,7 +72,58 @@ const GetAllFeed = async (request: IReq<IFeed>, response: IRes) => {
   }
 };
 
+const PostLikeDislikeById = async (request: IReq<IFeedPayload>, response: IRes) => {
+  try {
+    const { status } = request.body;
+    const userId = new Types.ObjectId(request.userId);
+    const postId = new Types.ObjectId(request.body.postId);
+    
+    if (status) {
+      const post = await FeedCollection.findById(postId);
+
+      if (!post?.userLikes.includes(userId)) {
```

We should check here, whether a post exists or not
Please apply condition like if(!post) then throw an error regarding not found feed